spec/service.spec.js

```diff
@@ -0,0 +1,325 @@
+const JokeService = require('../src/services/jokeService')
+const JokeContext = require('../src/context/index')
+const makeAxiosGetRequest = require('../src/services/axiosService')
```

Comment:
Also naming misalignment here
```suggestion
const axiosService = require('../src/services/axiosService')
```